src/service/execution/test.execution.service.ts

```diff
@@ -1,19 +1,17 @@
-import { Response } from '@angular/http';
-import { AuthHttp } from 'angular2-jwt';
 import { TestExecutionServiceConfig } from './test.execution.service.config';
 import { Injectable } from '@angular/core';
 import { TestExecutionState } from './test.execution.state';
+import { HttpClient } from '@angular/common/http';
 
 export interface TestExecutionStatus {
   path: string;
   status: TestExecutionState;
 }
 
 export abstract class TestExecutionService {
-  abstract execute(path: string): Promise<Response>;
+  abstract execute(path: string): Promise<any>;
```

Comment:
I'd vote for returning an empty object, or maybe a boolean (success, failure), instead of just removing the type information, but still passing back the complete HTTP response.

Reply:
I actually want to postpone this change when changing this to Observables. But I agree that we should get rid of Promise<any> types!